addons/website/views/website_templates.xml

```diff
@@ -574,42 +574,42 @@
         <content id="theme_customize_content_fonts" string="Fonts" title="Choose your fonts">
             <list string="Title">
                 <selection id="theme_customize_content_fonts_title">
-                    <opt data-xmlid="" data-font="1"/>
-                    <opt data-xmlid="website.option_font_title_02_variables" data-font="2"/>
-                    <opt data-xmlid="website.option_font_title_03_variables" data-font="3"/>
-                    <opt data-xmlid="website.option_font_title_04_variables" data-font="4"/>
-                    <opt data-xmlid="website.option_font_title_05_variables" data-font="5"/>
-                    <opt data-xmlid="website.option_font_title_06_variables" data-font="6"/>
+                    <opt string="Bootstrap" data-widget="auto" data-variable="headings-font-number" data-value="1" data-font="1"/>
```

Comment:
Why are you adding the "string" attribute? The name of the font appears automatically and if it is not the case, ask yourself why and fix that.

Reply:
In fact the name of the font still appears automatically but I have an error (on rendering). This said this is not probably related to `string` attribute but when I added it, it worked. Anyway, I forgot to check about this rendering error without this useless `string` attribute, so I'll fix that.